text/3-entrees/storage-api/sets-vecs-iteration.md

```diff
@@ -84,10 +99,24 @@ fn append_new_entries(origin) -> Result {
 }
 ```
 
-`append` encodes the new values, and pushes them to the already encoded vector without decoding the existing entries. This method removes the unnecessary steps for decoding and re-encoding the unchanged elements.
+`append` encodes the new values, and pushes them to the already encoded vector without decoding the
+existing entries. This method removes the unnecessary steps for decoding and re-encoding the
+unchanged elements.
 
 ## Iteration in the Runtime <a name = "iterate"></a>
 
-In general, iteration in the runtime should be avoided. *In the future*, [offchain-workers](https://github.com/substrate-developer-hub/recipes/issues/45) may provide a less expensive way to iterate over runtime storage items. Moreover, *[child tries](https://github.com/substrate-developer-hub/recipes/issues/35)* enable cheap inclusion proofs without the same lookup costs associated with vectors.
-
-Even so, there are a few tricks to alleviate the costs of iterating over runtime storage items like vectors. For example, it is [cheaper to iterate over a slice](https://twitter.com/heinz_gies/status/1121490424739303425) than a vector. With this in mind, store items in the runtime as vectors and transform them into slices after making storage calls. [3041](https://github.com/paritytech/substrate/pull/3041) introduced `insert_ref` and `put_ref` in order to allow equivalent reference-style types to be placed without copy (e.g. a storage item of `Vec<AccountId>` can now be written from a `&[AccountId]`). This enables greater flexibility when working with slices that are associated with vectors stored in the runtime.
+In general, iteration in the runtime should be avoided. _In the future_,
+[offchain-workers](https://github.com/substrate-developer-hub/recipes/issues/45) may provide a less
```

Comment:
Hmm, does this link need updating now?

Reply:
For sure. This is being overhauled in #272. That won't make it into this rc2 release though. :-/